docs/src/development/macos.md

```diff
@@ -6,7 +6,18 @@ Clone down the [Zed repository](https://github.com/zed-industries/zed).
 
 ## Dependencies
 
-- Install [Rust](https://www.rust-lang.org/tools/install)
+- Install [Rust](https://www.rust-lang.org/tools/install). If it's already installed, make sure it's up-to-date:
```

Comment:
```suggestion
- Install [rustup](https://www.rust-lang.org/tools/install). If it's already installed, make sure it's up-to-date:
```

Coming and looking at this again, neither `rustup update` nor `rustup target add` is needed: everything's handled automatically by `rustup` after any `cargo`-related command is run in tnis repo, due to https://github.com/zed-industries/zed/blob/main/rust-toolchain.toml file being present.

Let's remove all these lines as obsolete.